ansible/index.md

```diff
@@ -62,11 +73,9 @@ This documentation documents firmware version **2.0.0**. Changelogs and binaries
 
 ## Basics
 
-![Panel Overlay](images/ansible_CLEAN.png)
-
 Output and input functionality vary according to which application is running.
 
-Applications are determined by which USB device is plugged in. For example, when you plug in a Grid after a firmware update, it will be running Kria. Hold the preset key (next to the USB port) to switch to Meadowphysics; push it again to switch to Earthsea. Ansible will remember which application you were last running between power-ups.
+Applications are determined by which USB device is plugged in. For example, when you plug in a Grid after a firmware update, it will be running Kria. Hold the MODE key (next to the USB port) to switch to Meadowphysics; push it again to switch to Earthsea. Ansible will remember which application you were last running between power-ups.
```

Comment:
@rknLA : I wonder if *preset* isn't the more important identifier for that key? maybe it'd be better to include a note in the panel breakout that makes this explicit based on action:

`PRESET (press) / MODE (hold)`